dev/VSIX/Directory.Build.props

```diff
@@ -7,7 +7,7 @@
             https://pkgs.dev.azure.com/microsoft/ProjectReunion/_packaging/Project.Reunion.nuget.internal/nuget/v3/index.json
```

Comment:
Consider updating restore sources so that the build pipeline uses the ADO internal feed, but public builds use the version from nuget.org.

Reply:
I think this might already be happening as a fallback mechanism - the packages that were restored when building locally appeared to come from nuget.org.

I'm not exactly sure why that would have happened, though. And the package restore versions should have a better story that's a little more connected to the real-life versions, and aren't this hard-coded 0.5.7 business. I've opened a task to track this and try to make it better.

Reply:
Nuget.org is an upstream feed for our internal Nuget feed.